src/BSON/Value.stub.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/**
+ * @generate-class-entries static
+ * @generate-function-entries static
+ */
+
+namespace MongoDB\BSON;
+
+final class Value
```

Comment:
This class doesn't yet implement any interfaces, as I haven't thought about whether users should be able to serialise/unserialise these instances. Feedback is welcome.

Reply:
I'll leave this for later, as I'm not entirely sure what the BSON representation of a single value should be, as BSON always needs a root-level document.

Reply:
Oh, I thought we were talking about PHP serialization, not our BSON interfaces.

The advantage of using a BSON envelope for PHP serialization is that we can leverage existing APIs and BSON validation when unserializing. Not constructing a `bson_value_t` directly from, say, a base64 encoded string would also avoid potential attack vectors to create invalid BSON.

Reply:
Resolving this comment as `Value` implements `\Serializable` and serialisation has been added.